templates/init.html

```diff
@@ -1,7 +1,17 @@
+{% load staticfiles %}
+<script src="{% static "/static/js/my.js" %}"></script>
+
+<head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<script src="static/js/my.js"></script>
```

Comment:
Need to write `/static/js/my.js`

See this: [Another django project](https://github.com/kaustubhhiware/Foodspark/blob/master/foodspark/templates/foodspark/userdetails.html#L16)